sdk/communication/Azure.Communication.Common/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (Unreleased)
 
 ### Features Added
 
 ### Breaking Changes
+- Added support for a new communication identifier `MicrosoftBotIdentifier`.
```

Comment:
should be it under the added features?

Reply:
It's by ARB request, please see my comment https://github.com/Azure/azure-sdk-for-js/pull/24569#discussion_r1133783718

Reply:
Updated